test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -845,7 +845,7 @@ func (cont *NginxIngressController) Init() {
 	read := func(file string) string {
 		return string(testfiles.ReadOrDie(filepath.Join(IngressManifestPath, "nginx", file), ginkgo.Fail))
 	}
-	e2elog.Logf("initializing nginx ingress controller")
+	framework.Logf("initializing nginx ingress controller")
```

Comment:
Hmm... `e2elog.Logf` would be more appropriate!
```suggestion
	e2elog.Logf("initializing nginx ingress controller")
```